Swiftcord.xcodeproj/project.pbxproj

```diff
@@ -1383,7 +1383,7 @@
 				CURRENT_PROJECT_VERSION = 15;
 				DEAD_CODE_STRIPPING = YES;
 				DEVELOPMENT_ASSET_PATHS = "\"Swiftcord/Preview Content\"";
-				DEVELOPMENT_TEAM = Q382QLKDG3;
+				DEVELOPMENT_TEAM = 5Q2G7C7C3U;
```

Comment:
I think you should not including this🤔

Reply:
Yup, changes to signing and development team shouldn't be committed to ensure consistency